linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_finit/ansible/shared.yml

```diff
@@ -4,7 +4,7 @@
 # disruption = low
```

Comment:
Given https://github.com/ComplianceAsCode/content/commit/e7d99c03246ebc5eacba43156987e3548a6ea73e

Ensure that `multi_platform_ubuntu` is added